utils/defillama.ts

```diff
@@ -0,0 +1,33 @@
+import axios from "axios";
+
+const baseUrl = "https://api.llama.fi/protocol/uma-protocol";
+
+interface TvlData {
+  date: string;
+  totalLiquidityUSD: number;
+}
+
+export async function getDefillamaTvl() {
+  try {
+    const { data } = await axios.get(baseUrl);
```

Comment:
i would suggest a future optimization that maybe combines both latestTvl and percentChange in a single call, since it looks like this api call is quite heavy and we do it twice for the exact same data. not something though you need to optimize in this pr

Reply:
Yep, I agree. After this PR gets merged I will optimize to combine in a single call.